<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="header">
      <property name="styleSheet">
       <string notr="true">background-color:qlineargradient(spread:pad, x1:0.341, y1:0.034, x2:1, y2:0, stop:0 rgba(153, 0, 0, 245), stop:1 rgba(122, 17, 17, 246))</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Trueno</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>O-LIBRARY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Trueno</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>'PINJAM-PINJAM BUKU JADI MUDAH'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="body">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(170, 255, 0)rgb(85, 0, 0)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QFrame" name="judul_2">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 127)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Trueno</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:none;
color:rgb(170, 0, 0);
</string>
            </property>
            <property name="text">
             <string>LOGIN FORM</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#fff</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStatesMinorOutlyingIslands"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="inuser">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>120</y>
            <width>352</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="inpsw">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>220</y>
            <width>354</width>
            <height>41</height>
           </rect>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>359</x>
            <y>90</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto Bk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Username :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>193</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto Bk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Password :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="log_in">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>300</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background : rgb(165, 165, 165)</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
         <widget class="QPushButton" name="reg">
          <property name="geometry">
           <rect>
            <x>318</x>
            <y>340</y>
            <width>180</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background : rgb(165, 165, 165)</string>
          </property>
          <property name="text">
           <string>Don't Have Account? Register!</string>
          </property>
         </widget>
         <widget class="QLabel" name="error">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>270</y>
            <width>331</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 0, 0)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QFrame" name="footer">
      <property name="styleSheet">
       <string notr="true">background-color:qlineargradient(spread:pad, x1:0.341, y1:0.034, x2:1, y2:0, stop:0 rgba(153, 0, 0, 245), stop:1 rgba(122, 17, 17, 246))</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>O-Library Patch 0.0 beta</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
